src/Web/Larceny.hs

```diff
@@ -257,19 +260,36 @@ subs = M.fromList . map (\(x, y) -> (Blank x, y))
 -- | A plain text fill.
 --
 -- @
--- textFill "This text will be displayed in place of the blank"
+-- textFill "This text will be escaped and displayed in place of the blank"
 -- @
 textFill :: Text -> Fill s
 textFill t = textFill' (return t)
```

Comment:
worries me a bit that this isn't backwards compatible...can you convince me that this isn't dangerous?